diesel.toml

```diff
@@ -0,0 +1,9 @@
+# For documentation on how to configure this file,
+# see https://diesel.rs/guides/configuring-diesel-cli
+
+[print_schema]
+file = "src/schema.rs"
+custom_type_derives = ["diesel::query_builder::QueryId", "Clone"]
+
+[migrations_directory]
+dir = "/Users/joeykraut/work/fee-sweeper/migrations"
```

Comment:
I don't see any documentation for a `[migrations_directory]` section in the [link above](https://diesel.rs/guides/configuring-diesel-cli), but is it possible to have this be a relative path?

Reply:
yeah this was autogen, let me figure that out